cmd/setec/setec.go

```diff
@@ -441,6 +441,9 @@ func runPut(env *command.Env, name string) error {
 		return fmt.Errorf("failed to write secret: %w", err)
 	}
 	fmt.Printf("Secret saved as %q, version %d\n", name, ver)
+	if ver > 1 {
+		fmt.Printf("Be aware the default version of the secret is not automatically incremented. To make this version the default run setec activate %s %d\n", name, ver)
```

Comment:
Thanks for pointing this out.

Our goal was that this should be the _intended_ outcome, not a surprise. Rather than warning about it here, I'd like to improve the documentation so that this is less surprising.

The idea of separating "put" from "activate" was specifically that putting a new value _doesn't_ automatically expose the new value to your service immediately, but only after you activate it. That way you can roll out new versions (or roll back to old ones) with precise control.

Can you say more about where you would have liked to see that documented, for this not to be surprising?

Reply:
I guess in the put command but I still thing (because people often skim documentation) a warning here doesn't have too many downsides?  its not changing the behavior (which I understand the reasoning for) but that behavior is different from some other secrets stores people are used to.

Reply:
I agree, a warning here is also fine, but I'd like to do more than just that. Can you please take a look at #113 to see if that would address your concern?

Reply:
yeah looks good.